crates/consensus/tests/consensus_sim.rs

```diff
@@ -71,9 +71,9 @@ async fn consensus_simulation() {
 
         let wal_dir = temp_dir.clone();
         let handle = tokio::spawn(async move {
-            let mut current_height = 1;
+            let mut current_height = 0;
 
-            while current_height <= NUM_HEIGHTS {
+            while current_height <= NUM_HEIGHTS - 1 {
```

Comment:
Not super sure about the reason of this change. I'm assuming it's just you testing stuff?